app/vue.config.js

```diff
@@ -23,13 +22,7 @@ module.exports = {
 	},
 
 	configureWebpack: {
-		plugins: [
-			new WorkerPlugin(),
```

Comment:
No longer worth it?

Reply:
I decided to remove it in https://github.com/directus/directus/pull/5038/commits/63ff359944178c7ea31b6df4ea92c4e36f6f42be. The parsing isn't the bottleneck, and I only needed it because I wanted to display a million points (which took ~5s to parse on my laptop). Parsing ~100k points only locks the UI during half a sec.